src/styles/index.ts

```diff
@@ -4495,6 +4495,13 @@ const styles = (theme: ThemeColors) =>
             backgroundColor: theme.highlightBG,
         },
 
+        onboardingIconWrapper: {
+            alignItems: 'center',
+            justifyContent: 'center',
+            width: 40,
+            height: 40,
+        },
```

Comment:
Remove this style. It's not needed. We only need `padding: 10` for the `noneAccountingOption`

Reply:
@s77rt I updated.